pywinauto/base_wrapper.py

```diff
@@ -666,23 +665,27 @@ def click_input(
         if not absolute:
             coords = self.client_to_screen(coords)
 
-        _perform_click_input(button, coords, double, button_down, button_up,
-                             wheel_dist=wheel_dist, pressed=pressed,
-                             key_down=key_down, key_up=key_up)
-
+        message = None
         if use_log:
+            ctrl_text = self.window_text()
             if ctrl_text is None:
                 ctrl_text = six.text_type(ctrl_text)
-            message = 'Clicked ' + self.friendly_class_name() + ' "' + ctrl_text + \
-                      '" by ' + str(button) + ' button mouse click (x,y=' + \
-                      ','.join([str(coord) for coord in coords]) + ')'
-            if double:
-                message = 'Double-c' + message[1:]
             if button.lower() == 'move':
                 message = 'Moved mouse over ' + self.friendly_class_name() + \
-                          ' "' + ctrl_text + '" to screen point (x,y=' + \
-                          ','.join([str(coord) for coord in coords]) + ')'
-            ActionLogger().log(message)
+                          ' "' + ctrl_text + '" to screen point (x,y='
+            else:
+                message = 'Clicked ' + self.friendly_class_name() + ' "' + ctrl_text + \
+                          '" by ' + str(button) + ' button mouse click (x,y='
+                if double:
+                    message = 'Double-c' + message[1:]
+            message += ','.join([str(coord) for coord in coords]) + ')'
```

Comment:
One more thing I'd make better now: `message += str(tuple(coords))` (assuming to replace ` button mouse click (x,y=` to just ` mouse button at ` (see above).